lib/view_model/send/send_view_model.dart

```diff
@@ -327,13 +328,17 @@ abstract class SendViewModelBase with Store {
   void setFiatCurrency(FiatCurrency fiat) =>
       _settingsStore.fiatCurrency = fiat;
 
-  @action
-  void setSelectedCryptoCurrency(String cryptoCurrency) {
+  CryptoCurrency getCryptoCurrency(String cryptoCurrency) {
     try {
-      selectedCryptoCurrency = _wallet.balance.keys
-          .firstWhere((e) => cryptoCurrency.toLowerCase() == e.title.toLowerCase());
+      return _wallet.balance.keys.firstWhere(
+          (e) => cryptoCurrency.toLowerCase() == e.title.toLowerCase());
     } catch (e) {
-      selectedCryptoCurrency = _wallet.currency;
+      return _wallet.currency;
     }
   }
+
+  @action
+  void setSelectedCryptoCurrency(String cryptoCurrency) {
+    selectedCryptoCurrency = getCryptoCurrency(cryptoCurrency);
+  }
```

Comment:
**Minor:**
I don't see a need to have them as separate functions, just the old `setSelectedCryptoCurrency` would suffice